### PR TITLE
Astrolog register drop

### PR DESCRIPTION
**Pull request by listout**
Patch is also sent upstream, https://github.com/CruiserOne/Astrolog/pull/20

### CODE REVIEW

**Comment by listout** [comment]
Should be fixed in coming version https://github.com/CruiserOne/Astrolog/pull/20

File: app-misc/astrolog/astrolog-7.40-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Please bump with app-misc/astrolog-ephemeris
```
Comment (thesamesam) [inline app-misc/astrolog/astrolog-7.40-r1.ebuild:6]: ☝️

Comment (listout) [reply app-misc/astrolog/astrolog-7.40-r1.ebuild]: Done. Removed the patch from 7.40 ebuild and moved it into the 7.60 ebuild itself along with the version bumps.

**Review by juippis — APPROVED** [review]

File: app-misc/astrolog/astrolog-7.60.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		-i Makefile || die
+
+	# we use /usr/share/astrolog for config and (optional) ephemeris-data-files
+	sed -i -e "s:~/astrolog:/usr/share/astrolog:g" astrolog.h || die
+
+	# if we do NOT use X, we disable it by removing the -lX11 from the Makefile
+	# and remove the "#define X11" and "#define MOUSE" from astrolog.h
+	use X || ( sed -i -e "s:-lm -lX11:-lm:g" Makefile || die
+		   sed -i -e "s:#define X11:/*#define X11:g" astrolog.h || die
+		   sed -i -e "s:#define MOUSE:/*#define MOUSE:g" astrolog.h || die)
```
Comment (juippis) [inline app-misc/astrolog/astrolog-7.60.ebuild:46]: o.o

File: app-misc/astrolog/astrolog-7.60.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+EAPI=8
+
+# Please bump with app-misc/astrolog-ephemeris
+
+inherit toolchain-funcs
+
+DESCRIPTION="A many featured astrology chart calculation program"
+HOMEPAGE="https://www.astrolog.org/astrolog.htm"
+SRC_URI="https://www.astrolog.org/ftp/ast74src.zip"
```
Comment (arkamar) [inline app-misc/astrolog/astrolog-7.60.ebuild:12]: Just for the record, the `ast74src.zip` archive is for version 7.40 not 7.60.

**Pull request closed without merging**